src/sentry/integrations/jira/configure.py

```diff
@@ -74,8 +74,8 @@ def handle(self, request):
             form = JiraConfigForm(organizations, initial={'organizations': active_orgs})
             return self.get_response({'form': form})
 
-        enabled_orgs = form.cleaned_data['organizations']
-        disabled_orgs = list(set(o.id for o in organizations) - set(enabled_orgs))
+        enabled_orgs = [o for o in organizations if o.id in form.cleaned_data['organizations']]
```

Comment:
You've lost a unique filtering here.  Is `form.cleaned_data` going to be only unique orgs?

Reply:
yes it should be unique (but even if it weren't, I'm failing to see how that'd change things?)

Reply:
that method is idempotent, but I'm also *adding* a unique filter here because I'm selecting from the original set of orgs instead of the form